posthog/performance/sql.py

```diff
@@ -0,0 +1,185 @@
+"""https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry"""
```

Comment:
All other sql.py's are nested under posthog/models.

I think it makes sense to do the same here.